libs/test-cli/src/main.rs

```diff
@@ -98,8 +98,12 @@ async fn main() -> anyhow::Result<()> {
                 (None, false) => anyhow::bail!("{}", "either --stdin or --file-path is required".bold().red()),
             };
 
-            let api = migration_core::migration_api(&datamodel_string).await?;
+            migration_core::migration_api(&datamodel_string)
+                .await?
+                .reset(&())
+                .await?;
 
+            let api = migration_core::migration_api(&datamodel_string).await?;
```

Comment:
This is instantiating the migration engine twice, I would expect this to be super expensive if it happens thousands of times in a CI run. Why can't we reuse the first one?

Reply:
Although this isn't used for tests, we can forget about it.

Reply:
Oh, there was a bug in here I found! So it first resets the database, then runs the migrations but at this point we don't have the migration table anymore!

Reply:
oh ok — that CLI isn't unit-tested, so it broke with recent reset changes. LGTM!